app/src/main/java/org/p2p/wallet/home/repository/RefreshErrorRepository.kt

```diff
@@ -0,0 +1,8 @@
+package org.p2p.wallet.home.repository
+
+import kotlinx.coroutines.flow.Flow
+
+interface RefreshErrorRepository {
+    fun getFlow(): Flow<Unit>
+    fun notifyClick()
```

Comment:
Data layer doesn't know anything about clicks and stuff

Reply:
What do you mean? I've renamed (locally yet) getFlow -> getClickFlow

Reply:
In my opinion data layer can't use 'click' or UI-connected terms.  no clicks, only domain words and what it really means. 
we don't need to know about any clicks or other stuff on this level - we are interested in refresh and errors

fun notifyRefreshError - okay
fun getRefreshErrorFlow - okay
or smth else

Reply:
I see your point, but refreshError suggests that we're notifying about an error. In reality, the logic is about error resolution. So, let's rename 'click' to 'event' instead.

getRefreshEventFlow()
notifyEventRefreshed()

What do you think?

Reply:
Also the names of `RefreshError[Repository|Interactor]` could probably be changed to something more descriptive if their purpose is not clear.